uniffi_bindgen/src/scaffolding/templates/ExternalTypesTemplate.rs

```diff
@@ -0,0 +1,46 @@
+// Support for external types.
+
+// Types with an external `FfiConverter`...
+{% for (name, crate_name) in ci.iter_external_types() %}
+// `{{ name }}` is defined in `{{ crate_name }}`
+use {{ crate_name|crate_name_rs }}::FfiConverterType{{ name }};
+{% endfor %}
+
+// More complicated locally `Wrapped` types - we generate FfiConverter.
+{% for (name, prim) in ci.iter_wrapped_types() %}
+{% if loop.first %}
+
+// A trait that's in our crate for our external wrapped types to implement.
+trait UniffiCustomTypeWrapper {
+    type Wrapped;
+
+    fn wrap(val: Self::Wrapped) -> uniffi::Result<Self> where Self: Sized;
+    fn unwrap(obj: Self) -> Self::Wrapped;
+}
+
+{%- endif -%}
+
+// Type `{{ name }}` wraps a `{{ prim.canonical_name() }}`
+pub struct FfiConverterType{{ name }};
```

Comment:
Now that we're defining a couple things in the scaffolding code, I wonder if we should have a common prefix to namespace things.  Like maybe this should be named `UniffiFfiConverterType{{name}}`